### ISSUE TITLE
Tutanota rebranding to Tuta (Mail)

### ISSUE THREAD

**Issue by Elaborendum** [ferdium/ferdium-app#1445]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22enhancement+%3Asparkles%3A%22) for a feature request that matches the one I want to file, without success.

### Problem Description

Tutanota is [migrating](https://tuta.com/blog/tutanota-is-now-tuta) from tutanota.com to tuta.com. You can still login from tutanota.com, but you get a notice about the migration. I'm assuming it will fully migrate some day. Currently clicking the Tuta login link either opens `https://mail.tutanota.com/[object%20Object]` if logged in, resulting in nothing happening. Logging in through Tuta doesn't really make it the default either, since you of course get back to tutanota.com upon restart.

### Proposed Solution

Change url to `https://app.tuta.com/mail`, the new one.

### Alternatives Considered

Manually edit the recipe for my installation

**Comment by murilopereirame** [comment]
The ideal is, if the user doesn't have stored credentials, access directly the app.tuta.com, and if have credentials, handle the migration link correctly, so users that forgotten their passwords will be able to keep logged in at the new domain

**Comment by Elaborendum** [comment]
I'm not sure if I understand. If I login normally I think I remain in the old domain, only getting the prompt about the new domain at startup when the site is auto-logging me in. However, even if the service is set to open links in Ferdium, if I'm logged in it just opens `https://mail.tutanota.com/[object%20Object]` in a browser. If I'm not logged in, it lets me go to the new domain, but upon restart I just return to the old domain.

**Comment by murilopereirame** [comment]
The URL shown in the banner is used to migrate your credentials from the old domain (tutanota.com) to the new one (tuta.com), so, if a user has saved credentials at tutanota.com, is possible to move it to tuta.com. The solution must be handle the migration URL correctly (without this [object%20Object]), also is necessary to open the new URL inside Ferdium, because both websites needs to talk one with another.

In summary:
- New users/users without saved credentials should see the new page (app.tuta.com)
- Users with saved credentials in Ferdium should see the old page (mail.tutanota.com) and be able to migrate their credentials

I can take a look on it at weekend

**Comment by vraravam** [comment]
Is this still an issue? I see @rdwz's fix has been merged. Any working pending for this issue?

**Comment by mcmxcdev** [comment]
Closing due to inactivity, seems to be resolved for all participants
